app/lib/katello/errors.rb

```diff
@@ -35,6 +35,12 @@ def message
       end
     end
 
+    class EmptyBulkActionException < StandardError
+      def message
+        _("No registered Hosts found in selection.")
```

Comment:
we don't actually check the hosts to ensure katello-agent is installed on them.  If we were keeping these goferd-based actions around for a longer time, that might be a good strategy.  

What about 'No hosts registered with subscription-manager found in selection'?